tests/test_tracer.py

```diff
@@ -655,3 +656,24 @@ def task(t, errors):
     assert original_writer._trace_queue.qsize() == 1
     assert t.writer._trace_queue.qsize() == 1
     assert [[span]] == list(t.writer._trace_queue.get())
+
+
+def test_tracer_with_version():
+    t = ddtrace.Tracer()
+
+    # With global `config.version` defined
+    with BaseTracerTestCase.override_global_config(dict(version='1.2.3')):
```

Comment:
Do we also have the case covered where someone sets a global tag on the tracer _and_ has `DD_VERSION` available from the env vars?

Reply:
no, we don't, good catch

Reply:
I added a test case here, saying that `DD_VERSION` should take precedence